config/locales/en.yml

```diff
@@ -2647,13 +2647,31 @@ en:
           title: Select the municipality where you lived on December 31, %{filing_year}
       nj_review:
         edit:
+          amount: Amount
           county: County
           household_rent_own: Rent or own home
           municipality: Municipality
           primary_disabled: I have a disability
           property_tax_paid: Property taxes paid
           rent_paid: Rent paid
           spouse_disabled: My spouse has a disability
+          use_tax_applied: Consumer Use Tax Applied
+      nj_sales_use_tax:
+        edit:
+          followup_radio_label: Select one of the following
+          manual_use_tax_label: Enter calculated use tax. (Round to the nearest whole number)
+          subtitle_html: |
+            This includes:
+             <ul class="list--bulleted">
+               <li>Items purchased or services performed out-of-state, where there was no sales tax or the sales tax rate was below the NJ rate of 6.625%.</li>
+               <li>Online purchases where sales or use tax was not applied.</li>
+               <li>Purchases made by you or your spouse.</li>
+               <li><a href="https://nj.gov/treasury/taxation/su_10.shtml" target="_blank" rel="noopener noreferrer">Learn more</a>.
+             </ul>
```

Comment:
dust - is this the practice used throughout this repo, to include HTML in the localization strings directly? the alternative would be to interpolate just the text into HTML defined in `app/views/state_file/questions/nj_sales_use_tax/edit.html.erb` and then it would be the same HTML for `en`, `es`, etc. Probably not a problem in one case, but if it's the general practice it does sort of create a need to eventually have more E2E tests across all languages

Reply:
What do you mean? Like, having three separate strings for each list item and doing the `<li>` and `<ul>` in the erb file?

Funny because just a few days ago I gave the feedback to @mmazanec22 to consolidate li strings into an HTML like this https://github.com/codeforamerica/vita-min/pull/4869#discussion_r1805051551

This pattern (with the html) is what I've seen in the code elsewhere. It seems easier to manage than trying to keep several items straight. But we could talk about this and refactor across our codebase if there's some reason we shouldn't use this pattern?

Reply:
Yep my earlier comment is suggesting separate strings for each item so that there isn't HTML in the translation files. The reasons not to use this pattern would be to DRY the HTML, so refactors to it would only happen in one place. I think if it's the general practice we'd need to increase E2E tests across all languages since the code would be duplicated. Like for when we support 7 languages we'd have to make sure the HTML did not break for any of them. Thoughts?

Reply:
Let's leave as-is for now and discuss as a team during our thursday sync. Depending on the outcome we can do a follow-up refactor

Reply:
Yeah this is interesting. We do have this pattern in our codebase. I think I worry most about it being hard for the translators in Transifex to make changes to the text when its embedded in HTML but since historically it hasn't been an issue with them I would say its acceptable although I'm not sure preferred.

Reply:
at thursday sync we came for now to a pattern of putting HTML involving `class` or headers to be in the `.erb` for easier validation, but links and simple elements were acceptable in `.yml`